module/config.mjs

```diff
@@ -2344,18 +2344,57 @@ preLocalize("movementTypes", { sort: true });
 /*  Measurement                                 */
 /* -------------------------------------------- */
 
+/**
+ * @typedef {object} UnitConfiguration
+ * @property {string} label             Localized label for the unit.
+ * @property {string} abbreviation      Localized abbreviation for the unit.
+ * @property {number} conversion        Multiplier used to convert between various units.
+ * @property {string} [counted]         Localization path for counted plural forms in various unit display modes.
+ *                                      Only necessary if non-supported unit or using a non-standard name for a
+ *                                      supported unit.
+ * @property {string} [formattingUnit]  Unit formatting value as supported by javascript's internationalization system:
+ *                                      https://tc39.es/ecma402/#table-sanctioned-single-unit-identifiers. Only
+ *                                      required if the formatting name doesn't match the unit key.
+ * @property {string} type              Whether this is an "imperial" or "metric" unit.
+ */
+
 /**
  * The valid units of measure for movement distances in the game system.
  * By default this uses the imperial units of feet and miles.
  * @enum {string}
  */
 DND5E.movementUnits = {
-  ft: "DND5E.DistFt",
-  mi: "DND5E.DistMi",
-  m: "DND5E.DistM",
-  km: "DND5E.DistKm"
+  ft: {
+    label: "DND5E.UNITS.DISTANCE.Foot.Label",
+    abbreviation: "DND5E.UNITS.DISTANCE.Foot.Abbreviation",
+    conversion: 1,
+    formattingUnit: "foot",
+    type: "imperial"
+  },
+  mi: {
+    label: "DND5E.UNITS.DISTANCE.Mile.Label",
+    abbreviation: "DND5E.UNITS.DISTANCE.Mile.Abbreviation",
+    conversion: 5_280,
+    formattingUnit: "mile",
+    type: "imperial"
+  },
+  m: {
+    label: "DND5E.UNITS.DISTANCE.Meter.Label",
+    abbreviation: "DND5E.UNITS.DISTANCE.Meter.Abbreviation",
+    conversion: 3.281,
+    formattingUnit: "meter",
+    type: "metric"
+  },
+  km: {
+    label: "DND5E.UNITS.DISTANCE.Kilometer.Label",
+    abbreviation: "DND5E.UNITS.DISTANCE.Kilometer.Abbreviation",
+    conversion: 3_281,
```

Comment:
There might also be a simplified conversion for this but I don't know it off the top of my head. I will ask on discord.

Reply:
Well, if we do a 3,333 conversion, that will maintain the 1 km -> 1,000 m conversion and be the same relation of ft -> m and mi -> km.

Reply:
That sounds right, yeah